webknossos/webknossos/dataset/dataset.py

```diff
@@ -1355,6 +1366,31 @@ def _initialize_layer_from_properties(self, properties: LayerProperties) -> Laye
                 f"Failed to initialize layer: the specified category ({properties.category}) does not exist."
             )
 
+    @staticmethod
+    def get_remote_dataset_registry() -> Mapping[
+        str, RemoteOrganizationDatasetRegistry
+    ]:
```

Comment:
How about?
```suggestion
    @staticmethod
    def list_remote_datasets(organization_id: Optional[str] = None) -> RemoteOrganizationDatasetRegistry:
```

Reply:
Well, in combination with [my other comment](https://github.com/scalableminds/webknossos-libs/pull/763#discussion_r918960895), it essentially becomes a list